stations.csv

```diff
@@ -62235,13 +62235,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 66368;Gjøvik bussterminal;gjovik-bussterminal;;;60.79644;10.694396;;NO;Europe/Oslo;f;f;f;t;f;f;;;f;NSR|StopPlace|14744;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66368;
 66369;Eina bussterminal;eina-bussterminal;;;60.63065;10.597831;;NO;Europe/Oslo;f;f;f;t;f;f;;;f;NSR|StopPlace|14793;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66369;
 66370;Bleiken bussterminal;bleiken-bussterminal;;;60.46867;10.500407;;NO;Europe/Oslo;f;f;f;t;f;f;;;f;NSR|StopPlace|482;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66370;
-66371;Ancona Università;ancona-universita;;;43.586678;13.515875;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8313878;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66371;
-66372;Balanzano;balanzano;;;43.0795537;12.4221615;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8380601;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66372;
-66373;Bari Villaggio del Lavoratore;bari-villaggio-del-lavoratore;8311003;;41.104234;16.822596;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8311003;f;;f;;f;;f;;f;;f;;f;;f;8311003;t;S11003;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66373;
```

Comment:
how come you removed trenitalia_rtvt_id

Reply:
The trenitalia_rtvt_id is right at the end of the row `S11003`

If I don't remove it, the tests will fail saying there's a duplicate RAIL_ID